drivers/builtin/src/rsa.c

```diff
@@ -2645,14 +2645,13 @@ int mbedtls_rsa_pkcs1_verify(mbedtls_rsa_context *ctx,
                              const unsigned char *hash,
                              const unsigned char *sig)
 {
+#if defined(MBEDTLS_PKCS1_V21)
+    mbedtls_md_type_t mgf1_hash_id;
+#endif /* defined(MBEDTLS_PKCS1_V21) */
```

Comment:
I'm not sure I got the point in moving this declaration earlier. Unless it's strictly necessary, I suggest to keep changes at minimum.